test/validate.test.js

```diff
@@ -146,6 +146,16 @@ describe('validate.js', () => {
         assert.equal(paramsToValidate.source.url, "data:text/html;base64,PHA+VGhpcyBpcyBteSBjb250ZW50IGZyYWdtZW50LiBXaGF0J3MgZ29pbmcgb24/PC9wPgo=");
     });
 
+    it('validates parameters - watermark is a data uri', () => {
+        const paramsToValidate = {
+            watermarkContent: "data:text/html;base64,PHA+VGhpcyBpcyBteSBjb250ZW50IGZyYWdtZW50LiBXaGF0J3MgZ29pbmcgb24/PC9wPgo="
```

Comment:
It should be data:image/png. The image processing on the watermark content will be adamant to have the proper type. We probably could also try to validate type here for the data uri.

Reply:
See https://github.com/adobe/asset-compute-image-processing/blob/master/test/image.test.js#L177